src/backend/core/models.py

```diff
@@ -762,6 +762,12 @@ def get_abilities(self, user):
         can_update = (
             is_owner_or_admin or role == RoleChoices.EDITOR
         ) and not is_deleted
+        can_create_children = can_update and user.is_authenticated
+        can_destroy = (
+            is_owner
+            if self.is_root()
```

Comment:
I don't understand, why admin cannot delete root documents (and I don't see the related test)?

Reply:
This is the legacy behavior. And the related test is here https://github.com/suitenumerique/docs/blob/main/src/backend/core/tests/documents/test_api_documents_delete.py#L50